rest/server_context.go

```diff
@@ -292,8 +298,16 @@ func (sc *ServerContext) GetInactiveDatabase(ctx context.Context, name string) (
 			}
 		}
 	}
+	// handle the correct error message being returned for a corrupt database config
+	var httpErr *base.HTTPError
+	invalidConfig, ok := sc.invalidDatabaseConfigTracking.exists(name)
+	if !dbConfigFound && ok {
+		httpErr = base.HTTPErrorf(http.StatusNotFound, "Database %s config corrupt. Database config bucket name: %s not equal to bucket it is persisted in: %s. Must update database config immediately", name, invalidConfig.configBucketName, invalidConfig.persistedBucketName)
```

Comment:
I'd be inclined to have a single message about corruption, if the `fmt.Stringer` idea works that be great.

We want to add groupID and `base.MD`